recipe/conda_build_config.yaml

```diff
@@ -5,7 +5,7 @@ blas_impl:
 arpack:
   - 3.5
 boost:
-  - 1.65
+  - 1.65.1
```

Comment:
Is this really necessary?  Stuff built with the .1 revision is incompatible with stuff built with .0/.2?

If not, please avoid overspecifying this pin.

Reply:
Yes, sonames change between even patch version

Reply:
Ugh, that's terrible.  Boost needs to adapt accordingly in its run_exports - max_pin needs to be 'x.x.x'